diff -uNr statserial-1.1.orig/Makefile statserial-1.1/Makefile
--- statserial-1.1.orig/Makefile	Sun Dec 18 03:16:09 1994
+++ statserial-1.1/Makefile	Thu Dec 21 00:06:05 2000
@@ -7,13 +7,13 @@
 
 # for production code
 CFLAGS	= -Wall -O3 -fomit-frame-pointer
-LDFLAGS = -s -N
+LDFLAGS = 
 
 statserial:	statserial.o
-	$(LD) $(LDFLAGS) -o statserial statserial.o -lcurses
+	$(LD) $(LDFLAGS) -o statserial statserial.o -lncurses
 
 statserial.o: statserial.c
-	$(CC) $(CFLAGS) -c statserial.c
+	$(CC) $(CFLAGS) -c -I/usr/include/ncurses statserial.c
 
 install: statserial
 	install -m 555 statserial /usr/local/bin/statserial
